### PR TITLE
New options for identity_preserve

### PR DESCRIPTION
**Pull request by carchi8py**
##### SUMMARY
Add new an options for identity_preserve

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME
- na_ontap_snapmirror.py 

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by carchi8py** [comment]
@thedoubl3j how do we force ansibot to return this?

**Comment by thedoubl3j** [comment]
@carchi8py can either close/re-open or push an empty commit. Both retrigger CI.

File: lib/ansible/modules/storage/netapp/na_ontap_snapmirror.py [hunk 1]
```diff
@@ -349,6 +368,8 @@ def snapmirror_create(self):
             snapmirror_create.add_new_child('policy', self.parameters['policy'])
         if self.parameters.get('max_transfer_rate'):
             snapmirror_create.add_new_child('max-transfer-rate', str(self.parameters['max_transfer_rate']))
+        if self.parameters.get('identity_preserve'):
+            snapmirror_create.add_new_child('identity-preserve', str(self.parameters['identity_preserve']))
```
Comment (lonico) [inline lib/ansible/modules/storage/netapp/na_ontap_snapmirror.py:372]: do we need lowercase?

**Review by lonico — APPROVED** [review]
shipit

**Review by thedoubl3j — APPROVED** [review]
lgtm. small feature with test additions. shipit

**Pull request merged**